packages/botonic-cli/src/util/bot-config.ts

```diff
@@ -46,6 +63,49 @@ export class BotConfig {
         botonic_cli_version: botonicCliVersion,
       },
       packages,
+      tools: configLoaded.tools,
+      payloads: configLoaded.payloads,
+      webviews: configLoaded.webviews,
+    }
+  }
+  static async getTools(appDirectory: string) {
+    try {
+      const botConfigPath = path.join(appDirectory, '.src/bot-config.ts')
+      const output = await this.getOutputByCommand(
+        `npx ts-node ${botConfigPath}`
+      )
+      console.log('output', output)
+    } catch (err: any) {
+      console.error(`Error: ${err.message}`)
+    }
+    return []
+  }
+
+  static async loadBotConfig(appDirectory: string) {
+    try {
+      // Register ts-node to load TypeScript in real time
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      require('ts-node').register({
+        transpileOnly: true,
+        compilerOptions: {
+          module: 'commonjs',
+          target: 'es2018',
+          esModuleInterop: true,
+          allowSyntheticDefaultImports: true,
+        },
+      })
```

Comment:
put this ts config in an object so we can handle it better if we need to change it later